arch/risc-v/src/rv32m1/Make.defs

```diff
@@ -23,7 +23,7 @@
 HEAD_ASRC = rv32m1_vectors.S
 
 # Specify our general Assembly files
-CHIP_ASRCS = rv32m1_head.S riscv_syscall.S
```

Comment:
why remove?

Reply:
sys_call[x] series function on rv32im implemented in riscv_syscall.S, but in rv64gc they are inline functions in syscall.h